x-pack/plugins/security_solution/server/endpoint/routes/actions/audit_log_handler.ts

```diff
@@ -53,7 +78,16 @@ export const actionsLogRequestHandler = (
     }
 
     return res.ok({
-      body: result.body.hits.hits.map((e) => e._source),
+      body: {
+        total:
+          typeof result.body.hits.total === 'number'
+            ? result.body.hits.total
+            : result.body.hits.total.value,
+        items: result.body.hits.hits.map((e) => ({
```

Comment:
Have you tested what happens with no results? 

Or error cases? Or before the indices exist?

one thing to try:

clear out the data in ES. wipe and start anew. Don't perform any actions (or insert any action/response data). Just call this API right away. That should trigger some errors above since the actions and responses indices shouldn't exist yet. You can test the error handling that way. And then you need to probably add `ignore: [404]` to the es call above

Reply:
Good idea. I'll give that a try.

Reply:
done 856cefbb5f47bb17b95b55cd81c848c87549a4bf